moldesign/models/lammps_model.py

```diff
@@ -0,0 +1,280 @@
+# Copyright 2016 Autodesk Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import absolute_import
+
+import moldesign as mdt
+import moldesign.units as u
+
+from moldesign import compute
+from moldesign import forcefields as ff
+
+from .base import EnergyModelBase
+
+try:
+    from lammps import lammps, PyLammps
+    # WARNING: this is the real library, not our interface - this works because of absolute
+    # imports. We should probably rename this interface
+except ImportError:
+    print 'LAMMPS could not be imported; using remote docker container'
+    force_remote = True
+else:  # this should be configurable
+    force_remote = False  # debugging
+
+debugging = True # debugging
+
+import parmed as med
+import tempfile
+import os
+import sys
+from .amber2lammps import *
```

Comment:
`import *` is discouraged in most contexts because it makes it hard to track down variables.  This should be refactored to `from ..external import amber2lammps`, and all variables from that module should be referred to as `amber2lammps.Lammps` etc.